.github/workflows/build.yaml

```diff
@@ -91,14 +105,20 @@ jobs:
         run: cargo make --cwd ./examples build +${{ matrix.rust_toolchain }} --locked --profile ${{ matrix.cargo_profile }} --target ${{ matrix.target_triple.name }}
 
       - name: Package Examples (via Cargo Make)
+        # Skip this step when WDK 10.0.22621 is used with ARM64 runner as it doesn't ship certmgr and makecert binaries
+        if: ${{ !(matrix.wdk.version == '10.0.22621' && matrix.runner.arch == 'arm64') }}
```

Comment:
Note: I have noticed the package 10.0.22621 is missing certmgr and makecert  exes on arm64 machine when installed via winget channel. This `if` block is to skip the steps where these exes are needed.

Reply:
Yes, one if the recommended ways to do so is to set an env variable `CERT_TOOLS_PRESENT` and use it in the steps. I made that change now.